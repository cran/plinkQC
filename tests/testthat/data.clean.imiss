    FID     IID MISS_PHENO   N_MISS   N_GENO   F_MISS
  ID_26   ID_26          Y        0     8153        0
 ID_125  ID_125          Y        4     8153 0.0004906
 ID_169  ID_169          Y        6     8153 0.0007359
 ID_147  ID_147          Y        3     8153 0.000368
 ID_187  ID_187          Y        8     8153 0.0009812
  ID_17   ID_17          Y        8     8153 0.0009812
   ID_5    ID_5          Y        4     8153 0.0004906
  ID_74   ID_74          Y        4     8153 0.0004906
  ID_29   ID_29          Y        2     8153 0.0002453
 ID_163  ID_163          Y        7     8153 0.0008586
 ID_121  ID_121          Y        7     8153 0.0008586
 ID_164  ID_164          Y        7     8153 0.0008586
  ID_72   ID_72          Y        8     8153 0.0009812
  ID_19   ID_19          Y        9     8153 0.001104
 ID_119  ID_119          Y        1     8153 0.0001227
  ID_37   ID_37          Y        4     8153 0.0004906
 ID_115  ID_115          Y        3     8153 0.000368
  ID_30   ID_30          Y        3     8153 0.000368
  ID_40   ID_40          Y        5     8153 0.0006133
 ID_117  ID_117          Y        1     8153 0.0001227
 ID_114  ID_114          Y       13     8153 0.001595
  ID_49   ID_49          Y        9     8153 0.001104
 ID_170  ID_170          Y        6     8153 0.0007359
  ID_85   ID_85          Y        5     8153 0.0006133
  ID_15   ID_15          Y        1     8153 0.0001227
 ID_149  ID_149          Y        4     8153 0.0004906
 ID_141  ID_141          Y        2     8153 0.0002453
 ID_171  ID_171          Y       12     8153 0.001472
 ID_100  ID_100          Y        4     8153 0.0004906
 ID_111  ID_111          Y        2     8153 0.0002453
  ID_94   ID_94          Y        5     8153 0.0006133
 ID_165  ID_165          Y        3     8153 0.000368
 ID_131  ID_131          Y       10     8153 0.001227
 ID_142  ID_142          Y        6     8153 0.0007359
 ID_124  ID_124          Y        5     8153 0.0006133
  ID_68   ID_68          Y        5     8153 0.0006133
 ID_113  ID_113          Y        4     8153 0.0004906
 ID_188  ID_188          Y        3     8153 0.000368
 ID_189  ID_189          Y        4     8153 0.0004906
 ID_128  ID_128          Y        5     8153 0.0006133
   ID_2    ID_2          Y        3     8153 0.000368
   ID_1    ID_1          Y        9     8153 0.001104
 ID_166  ID_166          Y        4     8153 0.0004906
 ID_172  ID_172          Y        6     8153 0.0007359
 ID_145  ID_145          Y        1     8153 0.0001227
  ID_91   ID_91          Y        8     8153 0.0009812
 ID_199  ID_199          Y        6     8153 0.0007359
  ID_61   ID_61          Y        1     8153 0.0001227
 ID_104  ID_104          Y        8     8153 0.0009812
  ID_93   ID_93          Y        5     8153 0.0006133
  ID_56   ID_56          Y        3     8153 0.000368
  ID_21   ID_21          Y        2     8153 0.0002453
  ID_69   ID_69          Y        1     8153 0.0001227
  ID_65   ID_65          Y        3     8153 0.000368
 ID_182  ID_182          Y        6     8153 0.0007359
  ID_89   ID_89          Y        3     8153 0.000368
  ID_60   ID_60          Y        4     8153 0.0004906
  ID_36   ID_36          Y        7     8153 0.0008586
 ID_105  ID_105          Y        2     8153 0.0002453
 ID_109  ID_109          Y       15     8153  0.00184
  ID_87   ID_87          Y        4     8153 0.0004906
  ID_25   ID_25          Y       11     8153 0.001349
  ID_73   ID_73          Y        9     8153 0.001104
 ID_108  ID_108          Y        6     8153 0.0007359
 ID_190  ID_190          Y        2     8153 0.0002453
 ID_107  ID_107          Y       15     8153  0.00184
  ID_50   ID_50          Y        0     8153        0
  ID_67   ID_67          Y       11     8153 0.001349
  ID_92   ID_92          Y        8     8153 0.0009812
 ID_110  ID_110          Y        3     8153 0.000368
  ID_41   ID_41          Y       14     8153 0.001717
 ID_126  ID_126          Y        2     8153 0.0002453
  ID_51   ID_51          Y        1     8153 0.0001227
 ID_138  ID_138          Y        2     8153 0.0002453
  ID_33   ID_33          Y       13     8153 0.001595
  ID_66   ID_66          Y        8     8153 0.0009812
 ID_183  ID_183          Y        8     8153 0.0009812
 ID_136  ID_136          Y        2     8153 0.0002453
  ID_64   ID_64          Y        9     8153 0.001104
 ID_191  ID_191          Y        8     8153 0.0009812
 ID_195  ID_195          Y        5     8153 0.0006133
  ID_79   ID_79          Y        0     8153        0
 ID_148  ID_148          Y        3     8153 0.000368
 ID_135  ID_135          Y        2     8153 0.0002453
 ID_137  ID_137          Y        2     8153 0.0002453
  ID_27   ID_27          Y        1     8153 0.0001227
  ID_11   ID_11          Y        1     8153 0.0001227
 ID_140  ID_140          Y        3     8153 0.000368
 ID_143  ID_143          Y        3     8153 0.000368
  ID_81   ID_81          Y        4     8153 0.0004906
  ID_44   ID_44          Y        4     8153 0.0004906
 ID_184  ID_184          Y        4     8153 0.0004906
  ID_57   ID_57          Y        1     8153 0.0001227
  ID_83   ID_83          Y        2     8153 0.0002453
 ID_192  ID_192          Y        5     8153 0.0006133
 ID_173  ID_173          Y        4     8153 0.0004906
  ID_12   ID_12          Y        0     8153        0
 ID_174  ID_174          Y        8     8153 0.0009812
  ID_82   ID_82          Y        5     8153 0.0006133
  ID_38   ID_38          Y        4     8153 0.0004906
  ID_14   ID_14          Y        4     8153 0.0004906
  ID_13   ID_13          Y        5     8153 0.0006133
  ID_10   ID_10          Y        0     8153        0
 ID_196  ID_196          Y        6     8153 0.0007359
 ID_133  ID_133          Y       12     8153 0.001472
 ID_134  ID_134          Y        2     8153 0.0002453
 ID_118  ID_118          Y        0     8153        0
  ID_75   ID_75          Y        8     8153 0.0009812
  ID_24   ID_24          Y        7     8153 0.0008586
  ID_86   ID_86          Y        0     8153        0
  ID_22   ID_22          Y        4     8153 0.0004906
  ID_90   ID_90          Y       11     8153 0.001349
 ID_102  ID_102          Y        5     8153 0.0006133
 ID_151  ID_151          Y        6     8153 0.0007359
  ID_58   ID_58          Y        3     8153 0.000368
 ID_122  ID_122          Y        2     8153 0.0002453
  ID_78   ID_78          Y        3     8153 0.000368
  ID_18   ID_18          Y        7     8153 0.0008586
 ID_120  ID_120          Y        9     8153 0.001104
 ID_181  ID_181          Y        1     8153 0.0001227
 ID_144  ID_144          Y        4     8153 0.0004906
   ID_4    ID_4          Y        1     8153 0.0001227
  ID_53   ID_53          Y        4     8153 0.0004906
 ID_116  ID_116          Y        5     8153 0.0006133
 ID_176  ID_176          Y        4     8153 0.0004906
 ID_177  ID_177          Y        2     8153 0.0002453
  ID_63   ID_63          Y       14     8153 0.001717
  ID_70   ID_70          Y        7     8153 0.0008586
  ID_23   ID_23          Y       10     8153 0.001227
  ID_80   ID_80          Y        4     8153 0.0004906
 ID_193  ID_193          Y        4     8153 0.0004906
  ID_62   ID_62          Y       15     8153  0.00184
 ID_139  ID_139          Y        3     8153 0.000368
  ID_20   ID_20          Y        2     8153 0.0002453
  ID_77   ID_77          Y       16     8153 0.001962
  ID_39   ID_39          Y        0     8153        0
  ID_32   ID_32          Y        1     8153 0.0001227
   ID_9    ID_9          Y        3     8153 0.000368
  ID_31   ID_31          Y        0     8153        0
  ID_45   ID_45          Y        3     8153 0.000368
  ID_28   ID_28          Y        1     8153 0.0001227
  ID_55   ID_55          Y        0     8153        0
 ID_180  ID_180          Y       40     8153 0.004906
  ID_35   ID_35          Y        1     8153 0.0001227
 ID_123  ID_123          Y        3     8153 0.000368
  ID_96   ID_96          Y        6     8153 0.0007359
  ID_71   ID_71          Y        3     8153 0.000368
  ID_46   ID_46          Y        7     8153 0.0008586
 ID_197  ID_197          Y        5     8153 0.0006133
  ID_52   ID_52          Y        9     8153 0.001104
  ID_48   ID_48          Y        4     8153 0.0004906
 ID_112  ID_112          Y        6     8153 0.0007359
  ID_76   ID_76          Y        1     8153 0.0001227
 ID_198  ID_198          Y        2     8153 0.0002453
 ID_194  ID_194          Y        9     8153 0.001104
 ID_186  ID_186          Y        3     8153 0.000368
 ID_178  ID_178          Y       37     8153 0.004538
  ID_59   ID_59          Y        4     8153 0.0004906
   ID_3    ID_3          Y        5     8153 0.0006133
  ID_42   ID_42          Y        4     8153 0.0004906
 ID_103  ID_103          Y        3     8153 0.000368
 ID_132  ID_132          Y        3     8153 0.000368
  ID_97   ID_97          Y        2     8153 0.0002453
   ID_8    ID_8          Y       28     8153 0.003434
 ID_150  ID_150          Y        4     8153 0.0004906
  ID_16   ID_16          Y        5     8153 0.0006133
  ID_99   ID_99          Y       11     8153 0.001349
  ID_88   ID_88          Y       12     8153 0.001472
 ID_129  ID_129          Y        4     8153 0.0004906
  ID_34   ID_34          Y       14     8153 0.001717
  ID_54   ID_54          Y        9     8153 0.001104
 ID_106  ID_106          Y       11     8153 0.001349
 ID_130  ID_130          Y        5     8153 0.0006133
  ID_95   ID_95          Y       11     8153 0.001349
 ID_127  ID_127          Y        4     8153 0.0004906
 ID_200  ID_200          Y        5     8153 0.0006133
   ID_7    ID_7          Y        8     8153 0.0009812
  ID_98   ID_98          Y        2     8153 0.0002453
  ID_84   ID_84          Y        4     8153 0.0004906
 ID_101  ID_101          Y       12     8153 0.001472
