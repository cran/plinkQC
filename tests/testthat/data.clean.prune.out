rs3818646
rs7533
rs12140099
rs11121203
rs560514
rs12117402
rs11248979
rs7550997
rs6695883
rs636797
rs924267
rs4949444
rs12140988
rs1130800
rs6661037
rs6663355
rs2336240
rs7533470
rs7367713
rs2884796
rs11208830
rs11210834
rs11587504
rs10252
rs1008455
rs1416095
rs1461356
rs12143139
rs6664042
rs997791
rs4915899
rs2273122
rs637420
rs11208437
rs310219
rs34589269
rs4233429
rs4658265
rs1418460
rs1399285
rs6686057
rs3017627
rs1617980
rs10910823
rs6670984
rs12129691
rs2055975
rs4657219
rs2803881
rs760644
rs2032555
rs6693283
rs12132694
rs4651073
rs4287133
rs3904695
rs2146609
rs6428136
rs12068912
rs10464
rs2724384
rs11580728
rs950210
rs3766929
rs681239
rs592627
rs6691735
rs6429194
rs6686591
rs4430311
rs4854431
rs6755122
rs4669793
rs7562205
rs6738546
rs11886548
rs1453205
rs11691878
rs1401644
rs828259
rs1030278
rs17026712
rs1067405
rs4953161
rs17047952
rs13034020
rs1978656
rs3213747
rs3732238
rs2043095
rs12620252
rs367201
rs6542507
rs2553628
rs2683829
rs11688597
rs289834
rs6433371
rs13024527
rs11686762
rs11685862
rs7588230
rs2292387
rs1449636
rs9636284
rs1370661
rs7559258
rs6727573
rs1147169
rs1369843
rs4675231
rs3770976
rs1921738
rs1517347
rs4673839
rs12694574
rs1978859
rs11682403
rs2099728
rs3804784
rs1604809
rs11131063
rs6772477
rs2728939
rs1077524
rs17012829
rs6804277
rs6810058
rs17761091
rs9850057
rs4688424
rs3856591
rs9942038
rs9843168
rs6808687
rs9819574
rs1455832
rs2575775
rs1826637
rs6551495
rs4546169
rs7620600
rs9868829
rs825353
rs3914498
rs7616403
rs1554112
rs9818551
rs11917038
rs9289494
rs11707619
rs608841
rs4680107
rs3846078
rs6771155
rs9862554
rs16828327
rs1562465
rs17424987
rs1370086
rs7641571
rs9839536
rs6772982
rs2872247
rs7618021
rs2550295
rs2282766
rs2498323
rs3815189
rs4689440
rs17518096
rs2621250
rs13117296
rs17635673
rs2245395
rs2471712
rs16881804
rs1993059
rs1547525
rs6551519
rs10029192
rs1451180
rs12509018
rs6811350
rs9307061
rs6821061
rs4147542
rs2306478
rs951570
rs12502497
rs3113387
rs650483
rs7675351
rs17023204
rs13118340
rs11100393
rs4234956
rs4692779
rs2059516
rs6818447
rs16867538
rs16867617
rs2133761
rs153706
rs6884278
rs10076963
rs3828583
rs16877250
rs6452427
rs3913424
rs2967107
rs709360
rs2061476
rs3733939
rs31607
rs12152845
rs12187702
rs11167875
rs29822
rs256156
rs6899205
rs10454960
rs6937911
rs201233
rs530313
rs17430541
rs4711374
rs1018156
rs403606
rs2765943
rs9357421
rs1570360
rs7768852
rs9472807
rs1708552
rs9352202
rs9361677
rs310405
rs12193553
rs9445026
rs1338263
rs197870
rs761778
rs12661354
rs1931897
rs9402276
rs9403729
rs17084782
rs9479902
rs9365988
rs3778369
rs3823624
rs1062213
rs640945
rs6947510
rs6958567
rs4721380
rs801521
rs1178129
rs12699981
rs11764103
rs6462639
rs1962234
rs4947606
rs10808167
rs4732466
rs11978231
rs4148737
rs13229143
rs10265137
rs4727287
rs7812123
rs11764045
rs2299371
rs10251582
rs10954269
rs7793239
rs11984138
rs4875568
rs2627525
rs17070498
rs10109571
rs10104026
rs35349964
rs10109890
rs3750249
rs2904698
rs2188015
rs1910110
rs7463256
rs17051599
rs6557842
rs4872313
rs1481653
rs13260543
rs17526069
rs2279446
rs323352
rs987356
rs17769846
rs13259921
rs11781582
rs16937772
rs7464949
rs10156227
rs2912806
rs10958126
rs17680608
rs2250208
rs4370499
rs35798019
rs11778767
rs1347603
rs10109609
rs6988452
rs1793750
rs11779978
rs2738422
rs6989017
rs4736318
rs7025078
rs17800615
rs4740823
rs10815274
rs1345026
rs10812392
rs4879321
rs10814157
rs1566598
rs3088305
rs10758350
rs2253885
rs17556165
rs2811226
rs10991380
rs10818693
rs385057
rs696562
rs7870781
rs3793549
rs2809369
rs2041543
rs129889
rs11252026
rs7907906
rs2501686
rs12218816
rs12245880
rs2783592
rs703067
rs2153875
rs11009199
rs11812929
rs2503870
rs10740411
rs10994386
rs7076601
rs1227215
rs4128743
rs10996743
rs950054
rs7895740
rs7073334
rs1234888
rs2673467
rs4265510
rs1360116
rs7917710
rs4919044
rs2901783
rs1853208
rs2279508
rs1009184
rs12262953
rs12783584
rs719012
rs9919349
rs12763734
rs10500883
rs10835162
rs10835173
rs7115208
rs7115895
rs1488610
rs1875696
rs6591188
rs1193692
rs12281017
rs7943104
rs10501924
rs4754035
rs6591081
rs10789621
rs475115
rs11213219
rs7938457
rs10502143
rs2508490
rs4502006
rs559254
rs7111143
rs11222660
rs7131644
rs10791352
rs12818377
rs2878579
rs1894823
rs4387408
rs2048619
rs10842812
rs39635
rs12297206
rs10785669
rs2857671
rs1612103
rs12578814
rs11829757
rs970976
rs10878403
rs6581889
rs12424442
rs7961863
rs7306772
rs12370107
rs7312533
rs7295294
rs9651910
rs1549349
rs4765515
rs3129591
rs4768982
rs12428838
rs9566460
rs7332791
rs6561200
rs9568281
rs11616980
rs2322002
rs2169187
rs956922
rs7334564
rs356802
rs7321678
rs7988494
rs7991183
rs2760306
rs9519333
rs12861225
rs7991842
rs913397
rs4982421
rs10134120
rs2277459
rs1569898
rs2681728
rs17126141
rs17129201
rs11070339
rs2412779
rs599111
rs2620361
rs12595379
rs11071329
rs12898513
rs4432213
rs1976433
rs11856313
rs8039963
rs4778718
rs4338756
rs17541572
rs7171669
rs3760048
rs3785301
rs9933137
rs17135026
rs560941
rs182386
rs1967120
rs4148340
rs2380085
rs13339649
rs4967739
rs11643696
rs2896940
rs16967999
rs3743950
rs8054190
rs12596608
rs12919324
rs17682132
rs2386144
rs3816122
rs882704
rs12019079
rs6506586
rs8093156
rs10502476
rs2062420
rs1786604
rs16975000
rs495078
rs7351039
rs158867
rs4940955
rs9319760
rs1129763
rs10404882
rs11086065
rs403983
rs17798572
rs1820703
rs6510214
rs16970276
rs10426462
rs1997696
rs6056595
rs6078166
rs6135385
rs6131683
rs6131957
rs410488
rs6057673
rs17221823
rs7271624
rs6067191
rs2020975
rs4811780
rs6092911
rs2822446
rs9636746
rs2829931
rs6516749
rs2831981
rs468619
rs463021
rs9981339
rs2833446
rs2834257
rs11910387
rs9617661
rs5760440
rs13055470
rs11089436
rs7364186
rs5751239
rs5759125
rs2269606
rs4830438
rs7061322
rs5935233
rs5913960
rs1402078
rs1204396
rs4828044
rs2495684
rs5911410
rs1279807
